### PR TITLE
fix(searchbar): search.. dispears when clicked

### PR DESCRIPTION
**Pull request by Bensonre**
# Changes introduced

changed the font size like Carrie asked for,
fixed a bug where the first time search was clicked it doesn't disappear.

## Related issue(s):

- CSE-_user story or bug_
- CSE-_task number_

## Contributor checklist

- [ ] Wrote/updated tests for introduced changes
- [ ] Added new stories for created/modified components (if applicable)
- [ ] Checked Storybook for Accessibility issues (if applicable)
- [ ] ~~Updated Postman library for created/modified routes (if applicable)~~
- [ ] Verified that there are no issues in CircleCI
- [x] Assigned yourself to the PR
- [x] Removed `WIP:` from the PR title
- [x] Requested review from the SB Reviewers team

## Reviewer checklist

- [ ] Assigned yourself to the PR
- [ ] Read through the changes in the `Files changed` tab
- [ ] Verified that tests were written/modified
- [ ] Verified that stories were written/modified (if applicable)
- [ ] ~~Verified that Postman was updated (if applicable)~~
- [ ] Checked out the branch and tested changes locally
- [ ] Run storybook and checked that there are no accessibility issues

### CODE REVIEW

File: src/components/SearchBar/index.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export class SearchBar extends React.Component<SearchBarProps, SearchBarState> {
     super(props);
 
     this.state = {
-      text: props.initialText || 'Search...'
+      text: props.initialText || 'search...'
```
Comment (danvanhorn) [inline src/components/SearchBar/index.tsx]: This needs to be a capital S

Comment (Bensonre) [reply src/components/SearchBar/index.tsx]: okay then I messed up later in the code will fix

**Review by danvanhorn — DISMISSED** [review]
Everything works as expected, please address the comment that I made and then I'll approve this.

**Review by tnoelcke — APPROVED** [review]
Looks good!

**Pull request merged**